test/tests/parallel/set2/test_helper.bats

```diff
@@ -97,29 +101,35 @@ load 'test_helper/common'
   assert_success
 }
 
-@test "wait_for_finished_setup_in_container" {
+@test "${TEST_NAME_PREFIX} wait_for_finished_setup_in_container" {
+  # the container name is used here explicitly, because it cannot be a local
+  # variable as the `teardown` function would not have the correct scope
+  # and does not see the variable
```

Comment:
Only the `teardown()` needs the explicit value then? You could still use a variable for everywhere else that it is used?

Reply:
I think I actually did not write this comment, but it was in the file before. Maybe I can have a look later.